odoo/tools/config.py

```diff
@@ -218,6 +218,10 @@ def __init__(self, fname=None):
                          help='specify the SMTP username for sending email')
         group.add_option('--smtp-password', dest='smtp_password', my_default=False,
                          help='specify the SMTP password for sending email')
+        group.add_option('--smtp-ssl-certificate', dest='smtp_ssl_certificate', my_default=False,
```

Comment:
I do not really know, just adding the parameter there will update the `--help` command of odoo-bin

Reply:
I think we should update this documentation:
https://www.odoo.com/documentation/14.0/reference/cmdline.html#emails

What's the process ?

Reply:
> What's the process ?

Include the new doc in your change, it's in `doc/reference/cmdline.rst` (but beware because @AntoineVDV is going to break everything)

Reply:
@std-odoo We're going to move the doc from `odoo/odoo/doc` to `odoo/documentation` but you can already start writing your doc without worrying about that because the content will remain mostly the same.
Once we'll have moved the doc (beginning of next week) you'll be able to create a PR on `odoo/documentation` to documentation your changes. If you merge before us, we'll take care of everything.

Reply:
ok thank you :)